src/backend/firebase.ts

```diff
@@ -0,0 +1,13 @@
+import { initializeApp } from "firebase/app";
+import { getFirestore } from "firebase/firestore";
+
+const firebaseApp = initializeApp({
+  apiKey: "AIzaSyAwK8L-Z36JJoY0kGVP5K8eepI0SMG9wno",
```

Comment:
tutaj to nie ma problemu, ale generalnie klucze do api powinny być trzymane w lokalnym pliku .env i nie powinny wchodzić do gita - jest ryzyko, że ktoś je wykorzysta albo zacznie nabijać Ci tam jakieś dane i będziesz musiał płacić :D